toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractProjectRunner.java

```diff
@@ -152,11 +178,111 @@ protected void createAndOpenReport(final ITestPlan project) {
 			for (final ITestPage testPage : campaign.getTestCases()) {
 				String testPageHtmlReport = htmlReportGenerator.generatePageHtml(testPage);
 				htmlReportGenerator.writeFile(testPageHtmlReport, testPage.getName(), path);
+				System.out.println("\nNumber of Success step in test : " + testPage.getTestSuccessNumber() + "\n");
 			}
 		}
 
 		final String generatePageHtml = projectHtmlReportGenerator.generateProjectReportHtml(project);
 		this.projectHtmlReportGenerator.writeFile(generatePageHtml, pageName, path);
 		openReport(path, pageName);
 	}
-}
\ No newline at end of file
+
+	protected void sendLocalEmailReport (final ITestPlan project) {
+		for (final ICampaign campaign : project.getCampaigns()) {
+			String mailTo = "toto@talanlabs.com";
+			String mailFrom = "toast@gmail.com";
+			String host = "localhost";
+			Properties properties = System.getProperties();
+			properties.setProperty("mail.smtp.host", host);
+			Session session = Session.getDefaultInstance(properties);
+			int success = 0;
+			int failure = 0;
+			int total = 0;
+			try {
+				MimeMessage message = new MimeMessage(session);
+				message.setFrom(new InternetAddress(mailFrom));
+				message.addRecipient(Message.RecipientType.TO, new InternetAddress(mailTo));
+				message.setSubject("Test report");
+				StringBuilder sb = new StringBuilder();
+				
+				for (final ITestPage testPage : campaign.getTestCases()) {
+					success = testPage.getTestSuccessNumber();
+					failure = testPage.getTestFailureNumber();
+					total = success + failure;
+					String name = testPage.getName();
+					
+					sb.append("Test report for "+ name + "\n").append(System.lineSeparator());
+					sb.append("Total number of steps : " + total + "\n");
+					sb.append("Number of Success steps in test : " + success + "\nNumber of Failure steps in test : " + failure + "\n\n");
+
+				}
+				message.setText(sb.toString());
+				Transport.send(message);
+				System.out.println("Message sent successfully");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS106)